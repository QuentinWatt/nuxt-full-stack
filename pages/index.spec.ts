import { expect, test } from "vitest";
import index from "./index.vue";
import { mountSuspended } from "@nuxt/test-utils/runtime";

test("It renders hello world on the home page", async () => {
  const component = await mountSuspended(index);
  expect(component.html()).toContain("Hello world!");
});

test("It matches the snapshot", async () => {
  const component = await mountSuspended(index);
  expect(component.html()).matchSnapshot();
});
