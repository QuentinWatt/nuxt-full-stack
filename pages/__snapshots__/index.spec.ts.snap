// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`It matches the snapshot 1`] = `
"<main class="container mx-auto px-3 py-6">
  <h1 class="text-3xl font-bold underline"> Hello world! </h1>
</main>"
`;
